feathr-example/streaming-source-ingestion/eventhub_producer.py

```diff
@@ -0,0 +1,87 @@
+import io
+import json
+from datetime import datetime
+import os
+import avro.schema
+import numpy as np
+import pandas as pd
+import pytz
+from avro.io import BinaryEncoder, DatumWriter
+from confluent_kafka import Producer
+
+"""
+Produce some sample data for streaming feature using Kafka"""
+KAFKA_BROKER = "<EVENTHUB_HOST_NAME>:9093"
+KAFKA_TOPIC = "<EVENTHUB_TOPIC>"
+KAFKA_SASL_JAAS_CONFIG = "<EVENTHUB_SAS_POLICY_CONN_STRING>"
+
+GENERATION_SIZE_MIN = 1
+GENERATION_SIZE_MAX = 20
+
+os.environ['KAFKA_SASL_JAAS_CONFIG'] = KAFKA_SASL_JAAS_CONFIG
```

Comment:
Just curious what the purpose is of saving this to environment variable? I think the changes will be discarded after the script finishes?

Reply:
The previous implementation was using the environment util class, right now it is just passed directly as part of the config. Would be removing this.